runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -692,7 +692,44 @@ TR_J9ServerVM::getStaticReferenceFieldAtAddress(uintptrj_t fieldAddress)
 bool
 TR_J9ServerVM::stackWalkerMaySkipFrames(TR_OpaqueMethodBlock *method, TR_OpaqueClassBlock *clazz)
    {
+   if(!method)
+      {
+      return false;
+      }
+   
    JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+
+   if (_compInfoPT->getClientData()->getOrCacheVMInfo(stream)->_jlrMethodInvoke == NULL || 
```

Comment:
`getOrCacheVMInfo` is called several times. better to call it once and store the answer in a local reference.

Reply:
If `_jlrMethodInvoke` is NULL you have to send a message because it may have became non-null since the last time we checked.